src/tools/illink/test/Mono.Linker.Tests.Cases/Inheritance.Interfaces/OnReferenceType/UnusedComInterfaceIsRemoved.cs

```diff
@@ -0,0 +1,38 @@
+using System.Runtime.InteropServices;
+using Mono.Linker.Tests.Cases.Expectations.Assertions;
+using Mono.Linker.Tests.Cases.Expectations.Metadata;
+
+namespace Mono.Linker.Tests.Cases.Inheritance.Interfaces.OnReferenceType
+{
+	/// <summary>
+	/// With --keep-com-interfaces false, we apply the unused interface  rules also to com interfaces.
+	/// </summary>
+	[SetupLinkerArgument ("--keep-com-interfaces", "false")]
+	public class UnusedComInterfaceIsRemoved
+	{
+		public static void Main ()
+		{
+			var i = new A ();
+			i.Foo ();
+		}
+
+		interface IBar
```

Comment:
Declare as a COM interface?
```suggestion
		[ComImport]
		[Guid("04100298-3e0e-45cc-8e30-49539a900dd2")]
		interface IBar
```